### PR TITLE
add keyboard jaykeeb jk60

### PR DESCRIPTION
**Pull request by alabahuy**
## Description

Layout 60% support multi layout and keyboard case existing

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/jaykeeb/jk60/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT_60_ansi(
+        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
+        KC_LSFT,          KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,
+        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                             KC_RALT, KC_RGUI, KC_APP,  KC_RCTL
+    ),
+
+    [1] = LAYOUT_60_ansi(
```
Comment (waffle87) [inline keyboards/jaykeeb/jk60/keymaps/default/keymap.c:15]: This layer is not accessible.

File: keyboards/jaykeeb/jk60/matrix_diagram.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Matrix Diagram for Jaykeeb JK65
```
Comment (waffle87) [inline keyboards/jaykeeb/jk60/matrix_diagram.md:1]: This is for a different keyboard?

**Review by waffle87 — CHANGES_REQUESTED** [review]

File: keyboards/jaykeeb/jk60/matrix_diagram.md (hunk 2 above)
Comment (alabahuy) [reply keyboards/jaykeeb/jk60/matrix_diagram.md:1]: sorry, forgot to fix it

File: keyboards/jaykeeb/jk60/keymaps/default/keymap.c (hunk 1 above)
Comment (alabahuy) [reply keyboards/jaykeeb/jk60/keymaps/default/keymap.c:15]: oke add keymap layer

**Review by waffle87 — APPROVED** [review]

File: keyboards/jaykeeb/jk60/keyboard.json [hunk 3]
```diff
@@ -0,0 +1,447 @@
...
+        "on_state": 0
+    },
+    "processor": "RP2040",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x7760",
+        "vid": "0x414C"
+    },
+    "community_layouts": ["60_ansi", "60_ansi_split_bs_rshift", "60_ansi_tsangan", "60_ansi_wkl", "60_ansi_wkl_split_bs_rshift"],
```
Comment (dunk2k) [inline keyboards/jaykeeb/jk60/keyboard.json:30]:
```suggestion
    "community_layouts": ["60_ansi", "60_ansi_split_bs_rshift", "60_ansi_tsangan", "60_tsangan_hhkb", "60_ansi_wkl", "60_ansi_wkl_split_bs_rshift", "60_hhkb"],
```

File: keyboards/jaykeeb/jk60/keyboard.json [hunk 4]
```diff
@@ -0,0 +1,447 @@
...
+                {"label": "Shift", "matrix": [3, 12], "x": 12.25, "y": 3, "w": 1.75},
+                {"label": "Fn", "matrix": [3, 13], "x": 14, "y": 3},
+
+                {"label": "Ctrl", "matrix": [4, 0], "x": 0, "y": 4, "w": 1.5},
+                {"label": "Alt", "matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.5},
+                {"label": "Space", "matrix": [4, 6], "x": 4, "y": 4, "w": 7},
+                {"label": "Alt", "matrix": [4, 10], "x": 11, "y": 4, "w": 1.5},
+                {"label": "Ctrl", "matrix": [4, 13], "x": 13.5, "y": 4, "w": 1.5}
+            ]
+        }
```
Comment (dunk2k) [inline keyboards/jaykeeb/jk60/keyboard.json:445]:
```suggestion
        },
        "LAYOUT_60_hhkb": {
            "layout": [
                {"label": "Esc", "matrix": [0, 0], "x": 0, "y": 0},
                {"label": "1", "matrix": [0, 1], "x": 1, "y": 0},
                {"label": "2", "matrix": [0, 2], "x": 2, "y": 0},
                {"label": "3", "matrix": [0, 3], "x": 3, "y": 0},
                {"label": "4", "matrix": [0, 4], "x": 4, "y": 0},
                {"label": "5", "matrix": [0, 5], "x": 5, "y": 0},
                {"label": "6", "matrix": [0, 6], "x": 6, "y": 0},
                {"label": "7", "matrix": [0, 7], "x": 7, "y": 0},
                {"label": "8", "matrix": [0, 8], "x": 8, "y": 0},
                {"label": "9", "matrix": [0, 9], "x": 9, "y": 0},
                {"label": "0", "matrix": [0, 10], "x": 10, "y": 0},
                {"label": "-", "matrix": [0, 11], "x": 11, "y": 0},
                {"label": "=", "matrix": [0, 12], "x": 12, "y": 0},
                {"label": "Backspace", "matrix": [0, 13], "x": 13, "y": 0},
                {"label": "Delete", "matrix": [1, 13], "x": 14, "y": 0},

                {"label": "Tab", "matrix": [1, 0], "x": 0, "y": 1, "w": 1.5},
                {"label": "Q", "matrix": [1, 1], "x": 1.5, "y": 1},
                {"label": "W", "matrix": [1, 2], "x": 2.5, "y": 1},
                {"label": "E", "matrix": [1, 3], "x": 3.5, "y": 1},
                {"label": "R", "matrix": [1, 4], "x": 4.5, "y": 1},
                {"label": "T", "matrix": [1, 5], "x": 5.5, "y": 1},
                {"label": "Y", "matrix": [1, 6], "x": 6.5, "y": 1},
                {"label": "U", "matrix": [1, 7], "x": 7.5, "y": 1},
                {"label": "I", "matrix": [1, 8], "x": 8.5, "y": 1},
                {"label": "O", "matrix": [1, 9], "x": 9.5, "y": 1},
                {"label": "P", "matrix": [1, 10], "x": 10.5, "y": 1},
                {"label": "[", "matrix": [1, 11], "x": 11.5, "y": 1},
                {"label": "]", "matrix": [1, 12], "x": 12.5, "y": 1},
                {"label": "\\", "matrix": [2, 13], "x": 13.5, "y": 1, "w": 1.5},

                {"label": "Caps Lock", "matrix": [2, 0], "x": 0, "y": 2, "w": 1.75},
                {"label": "A", "matrix": [2, 1], "x": 1.75, "y": 2},
                {"label": "S", "matrix": [2, 2], "x": 2.75, "y": 2},
                {"label": "D", "matrix": [2, 3], "x": 3.75, "y": 2},
                {"label": "F", "matrix": [2, 4], "x": 4.75, "y": 2},
                {"label": "G", "matrix": [2, 5], "x": 5.75, "y": 2},
                {"label": "H", "matrix": [2, 6], "x": 6.75, "y": 2},
                {"label": "J", "matrix": [2, 7], "x": 7.75, "y": 2},
                {"label": "K", "matrix": [2, 8], "x": 8.75, "y": 2},
                {"label": "L", "matrix": [2, 9], "x": 9.75, "y": 2},
                {"label": ";", "matrix": [2, 10], "x": 10.75, "y": 2},
                {"label": "'", "matrix": [2, 11], "x": 11.75, "y": 2},
                {"label": "Enter", "matrix": [2, 12], "x": 12.75, "y": 2, "w": 2.25},

                {"label": "Shift", "matrix": [3, 0], "x": 0, "y": 3, "w": 2.25},
                {"label": "Z", "matrix": [3, 2], "x": 2.25, "y": 3},
                {"label": "X", "matrix": [3, 3], "x": 3.25, "y": 3},
                {"label": "C", "matrix": [3, 4], "x": 4.25, "y": 3},
                {"label": "V", "matrix": [3, 5], "x": 5.25, "y": 3},
                {"label": "B", "matrix": [3, 6], "x": 6.25, "y": 3},
                {"label": "N", "matrix": [3, 7], "x": 7.25, "y": 3},
                {"label": "M", "matrix": [3, 8], "x": 8.25, "y": 3},
                {"label": ",", "matrix": [3, 9], "x": 9.25, "y": 3},
                {"label": ".", "matrix": [3, 10], "x": 10.25, "y": 3},
                {"label": "/", "matrix": [3, 11], "x": 11.25, "y": 3},
                {"label": "Shift", "matrix": [3, 12], "x": 12.25, "y": 3, "w": 1.75},
                {"label": "Fn", "matrix": [3, 13], "x": 14, "y": 3},

                {"label": "GUI", "matrix": [4, 1], "x": 1.5, "y": 4},
                {"label": "Alt", "matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.5},
                {"label": "Space", "matrix": [4, 6], "x": 4, "y": 4, "w": 7},
                {"label": "Alt", "matrix": [4, 10], "x": 11, "y": 4, "w": 1.5},
                {"label": "GUI", "matrix": [4, 11], "x": 12.5, "y": 4}
            ]
        }
```

**Review by dunk2k — CHANGES_REQUESTED** [review]
add remaining compatible Community Layout support

**Comment by alabahuy** [comment]
> add remaining compatible Community Layout support

oke done @dunk2k

**Review by drashna — APPROVED** [review]

**Pull request merged**